samples/bot-conversation-sso-quickstart/js/dialogs/logoutDialog.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityTypes } = require('botbuilder');
+const { ComponentDialog } = require('botbuilder-dialogs');
+
+class LogoutDialog extends ComponentDialog {
```

Comment:
Do we have enough comment as Tab SSO template to explain what the code is doing for user?

Reply:
Looks good for me.